.travis.yml

```diff
@@ -45,4 +46,4 @@ install:
 script:
   - if [[ $BUILD == TEST ]]; then make test; fi
   - if [[ $BUILD == COVERAGE ]]; then make coverage; fi
-  - if [[ $BUILD == DOCS ]]; then cd doc; sphinx-build -W -a -E -b html -d _build/doctrees . _build/html; fi
+  - if [[ $BUILD == DOCS ]]; then make doc; fi
```

Comment:
I think the `-W` makes the docs build fail if there is some problem, but `make doc` doesn't.
It's useful to catch docs build issues that way.
You could add a `make doc-ci` or find some other way to keep the CI behaviour if you agree.